wnfs-wasm/src/fs/private/directory.rs

```diff
@@ -166,11 +166,7 @@ impl PrivateDirectory {
                 .await
                 .map_err(error("Cannot get node"))?;
 
-            Ok(utils::create_private_op_result(
-                directory,
-                forest,
-                result.map(PrivateNode),
-            )?)
```

Comment:
:+1: Let's make sure that we capture this change somewhere in the CHANGELOG, so that once @icidasset or someone upgrades to the next rs-wnfs version they're not surprised why it crashes again :sweat_smile: